lib/install/validate/storage.go

```diff
@@ -31,6 +31,11 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/datastore"
 )
 
+const (
+	nfsScheme = "nfs"
```

Comment:
yes I originally did this to avoid pulling in a lot of packages. I will look at the common package